nex/register_common_secure_server_protocols.go

```diff
@@ -0,0 +1,30 @@
+package nex
+
+import (
+	"github.com/PretendoNetwork/mario-kart-7/globals"
+	matchmake_extension "github.com/PretendoNetwork/nex-protocols-common-go/matchmake-extension"
+	matchmaking "github.com/PretendoNetwork/nex-protocols-common-go/matchmaking"
+	matchmaking_ext "github.com/PretendoNetwork/nex-protocols-common-go/matchmaking-ext"
```

Comment:
```suggestion
	match_making_ext "github.com/PretendoNetwork/nex-protocols-common-go/matchmaking-ext"
```

Just for consistency. This is what the module is named in common